ts/components/leftpane/overlay/OverlayClosedGroup.tsx

```diff
@@ -85,12 +123,14 @@ export const OverlayClosedGroup = () => {
       <SessionSpinner loading={loading} />
 
       <SpacerLG />
-      <div className="group-member-list__container">
+      <SessionSearchInput />
+
+      <StyledGroupMemberListContainer>
         {noContactsForClosedGroup ? (
-          <div className="group-member-list__no-contacts">{window.i18n('noContactsForGroup')}</div>
+          <NoContacts />
         ) : (
           <div className="group-member-list__selection">
-            {privateContactsPubkeys.map((memberPubkey: string) => (
+            {contactsToRender.map((memberPubkey: string) => (
```

Comment:
Doesn't this need a key prop?

Reply:
it's right there, line 137

Reply:
it's right there, 3 lines below, line 137